Makefile

```diff
@@ -1,5 +1,3 @@
-QC_SOC_TARGET?="APQ8074"
-
 all: linux qurt rpi bebop edison
```

Comment:
`QC_SOC_TARGET` is still there, it's just moved to here:
https://github.com/PX4/DriverFramework/blob/2ec96d5ee25bcfa5ddbd10066e39578189114a50/cmake/toolchains/Toolchain-qurt.cmake#L37